docs/getting_started.md

```diff
@@ -60,6 +60,8 @@ The variable group should contain the following required variables:
 | WORKSPACE_NAME           | mlops-AML-WS             |
 | WORKSPACE_SVC_CONNECTION | aml-workspace-connection |
 | ACI_DEPLOYMENT_NAME      | diabetes-aci             |
+| DATASET_NAME      | diabetes_training             |
+| MODEL_NAME      | diabetes_model.pkl             |
```

Comment:
`MODEL_NAME` is already provided in the variables.yml and we're trying to minimize manual entry of variables to the group.  Please remove.

Reply:
See update

Reply:
Code looks good, but we should only be putting values here which are not required to be manually entered.

Reply:
I updated the getting started markdown to remove MODEL_NAME and DATASET_NAME. Is there something else that needs to be removed?

Reply:
I removed the unused DATA vars from the ENV example

Reply:
and from env-variables.py